[2024-09-06T00:08:30.065+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-06T00:08:30.086+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_dag.map_sequia_indicator manual__2024-09-05T23:38:26.329805+00:00 [queued]>
[2024-09-06T00:08:30.101+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_dag.map_sequia_indicator manual__2024-09-05T23:38:26.329805+00:00 [queued]>
[2024-09-06T00:08:30.102+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-09-06T00:08:30.218+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): map_sequia_indicator> on 2024-09-05 23:38:26.329805+00:00
[2024-09-06T00:08:30.223+0000] {warnings.py:112} WARNING - /usr/local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=883) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-06T00:08:30.225+0000] {standard_task_runner.py:72} INFO - Started process 885 to run task
[2024-09-06T00:08:30.226+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_dag', 'map_sequia_indicator', 'manual__2024-09-05T23:38:26.329805+00:00', '--job-id', '206', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl_dag.py', '--cfg-path', '/tmp/tmph0kv6_qe']
[2024-09-06T00:08:30.228+0000] {standard_task_runner.py:105} INFO - Job 206: Subtask map_sequia_indicator
[2024-09-06T00:08:30.815+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_dag.map_sequia_indicator manual__2024-09-05T23:38:26.329805+00:00 [running]> on host da9f5d3bcb59
[2024-09-06T00:08:31.001+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_dag' AIRFLOW_CTX_TASK_ID='map_sequia_indicator' AIRFLOW_CTX_EXECUTION_DATE='2024-09-05T23:38:26.329805+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-05T23:38:26.329805+00:00'
[2024-09-06T00:08:31.002+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-06T00:08:31.212+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-06T00:08:31.224+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-06T00:08:31.225+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_dag, task_id=map_sequia_indicator, run_id=manual__2024-09-05T23:38:26.329805+00:00, execution_date=20240905T233826, start_date=20240906T000830, end_date=20240906T000831
[2024-09-06T00:08:31.393+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-06T00:08:31.451+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-06T00:08:31.592+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
